dexter/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,7 @@
   -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.karumi.dexter"
+    android:allowBackup="true"
```

Comment:
Isn't this the default value for ``allowBackup``? Should we add this param to the library manifest? Couldn't this be a problem during the merge manifest process?